odra-casper/backend/src/lib.rs

```diff
@@ -1,4 +1,8 @@
+#![no_std]
 //! Generate Casper contract and interact with Casper host.
+#![cfg_attr(not(test), feature(core_intrinsics))]
```

Comment:
When is it used for tests?

Reply:
Panic handler requires this feature, and the feature is active NOT in test.